### ISSUE TITLE
Focus Vertical, Screenreader Doesn't Read the Content of the Tabs

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#604]
![image](https://user-images.githubusercontent.com/7510842/33373141-dd3dd6e0-d500-11e7-9557-5a323aecf755.png)

- [x]  Please check the vertical version 
https://swiss.github.io/styleguide/en/teaser.html#05-teaser-02-focus-vertical

- [x]  And check the horizontal version as well
https://swiss.github.io/styleguide/en/teaser.html#05-teaser-01-focus

**Comment by gillerr** [comment]
The tab selection with Enter key seems to be currently broken

**Comment by gillerr** [comment]
ChromeVox breaks the tab selection with Enter key

**Comment by bit-pro-iew-eui** [comment]
Validated, content accessible
